packages/assert/README.md

```diff
@@ -1,10 +1,50 @@
-assert
+@0xproject/assert
 ------
 
 Standard type and schema assertions to be used across all 0x projects and packages
 
-## Install
+## Installation
 
 ```bash
-npm install @0xproject/assert --save
+yarn add @0xproject/assert
+```
+
+## Usage
+
+TODO - Custom documentation page is in progress.
```

Comment:
Mh... don't think we want a docs page for `assert` package. Thoughts @abandeali1 @BMillman19? I feel like we should only have dedicated docs pages for tools we expect third-parties to use.